src/apps/tools/OswAppToolsApp.cpp

```diff
@@ -0,0 +1,26 @@
+#include "./apps/tools/OswAppToolsApp.h"
+
+#include <config.h>
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_config.h>
+#include <osw_config_keys.h>
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include <services/OswServiceTaskWebserver.h>
+#include <services/OswServiceTaskWiFi.h>
+#include <services/OswServiceTasks.h>
+
+void OswAppToolsApp::setup() {}
+
+void OswAppToolsApp::loop() {
+    OswHal* hal = OswHal::getInstance();
+    hal->gfx()->setTextSize(2);
+
+    hal->gfx()->setTextCursor(40,60);
+    hal->gfx()->print("Tools Page 1");
```

Comment:
Also, there is nothing in the added Tools.

Reply:
The desktop git client doesn't seem to be working right, I will try to get a commit to work with git with the develop branch..